features/lwipstack/lwip/src/include/lwip/opt.h

```diff
@@ -674,7 +674,8 @@
  * that returns 1 to accept a packet or 0 to drop a packet.
  */
 #if !defined ETHARP_SUPPORT_VLAN || defined __DOXYGEN__
-#define ETHARP_SUPPORT_VLAN             0
+#define ETHARP_SUPPORT_VLAN             1
```

Comment:
Is is not enough by defining these in the mbed os config?

Reply:
No, the two functions CHECK and SET are mandatory as well. Check is going to be called by ethernet_input to see if we accept or drop the packet, and Set is used by ethernet_output to set the TCI field of the VLAN header.